README.md

```diff
@@ -80,32 +81,12 @@ Additionally, our initial design requirements document can be found [here](docs/
 
 ### [CHANGELOG](CHANGELOG.md)
 
-For a history of changes to HELICS
+A history of changes to HELICS
 
 ### [ROADMAP](docs/ROADMAP.md)
 
-For a snapshot of some current plans for what is to come.
-
-## Installation
-
-### Windows
-
-Each [release](https://github.com/GMLC-TDC/HELICS/releases/latest) comes with a Windows installer and zip file containing the HELICS apps and C shared library with Python 3.6 and Java 1.8 interfaces, zip files containing compiled HELICS static libraries built against several MSVC versions, and an archive containing just the C shared library with headers. For building with Debug support it is generally recommended to build from Source or use HELICS as a subproject.
-Make sure the latest [Visual C++ Redistributable](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads) is installed.
-
-### Cross-platform Python (via pip)
-
-Support for installing the Python interface and HELICS apps is provided with pip, see [pip install](https://python.helics.org) for the instructions. The files used to build the pip package are in a separate [repo](https://github.com/GMLC-TDC/pyhelics).
-
-### Linux and macOS (via Spack)
-
-HELICS can be installed on Linux (and macOS) using Spack, a package manager aimed at HPC environments. See [spack install](https://helics.readthedocs.io/en/latest/installation/package_manager.html#install-using-spack-macos-linux) for the instructions.
+A snapshot of some current plans for what is to come.
 
-### Building from Source
-
-- [Windows](https://docs.helics.org/en/latest/user-guide/installation/windows.html)
-- [Mac](https://docs.helics.org/en/latest/user-guide/installation/mac.html)
-- [Ubuntu](https://docs.helics.org/en/latest/user-guide/installation/linux.html)
 
 ### Docker
```

Comment:
This section looks like it should be removed (moved to the installation guide on RTD), moved up a level to `## Docker`, or have the `## Installation` header left in (maybe with a quick word and a link to the installation guide to make install instructions more prominent).

The table of contents at the top of the README also needs updating to reflect the changes.